eng/testing/runsettings.targets

```diff
@@ -26,6 +26,9 @@
   <Target Name="GenerateRunSettingsFile"
           DependsOnTargets="SetupCoverageFilter">
     <PropertyGroup>
+      <RunSettingsDotNetHostPath Condition="'$(RunSettingsDotNetHostPath)' == ''">$(NetCoreAppCurrentTestHostPath)$([System.IO.Path]::GetFileName('$(DotNetTool)'))</RunSettingsDotNetHostPath>
+      <RunSettingsTestCaseFilter Condition="'$(RunSettingsTestCaseFilter)' == ''">$(_testFilter)</RunSettingsTestCaseFilter>
```

Comment:
The `_testFilter` property above should probably also be conditioned on if RunSettingsTestCaseFilter is not set.